src/test/integration/api_request.py

```diff
@@ -0,0 +1,53 @@
+import json
+import requests
+import urllib2
+import ssl
+from configuration_load import *
+
+
+def good_request():
+    query_params = {'q': 'Oxford'}
+    headers = {'Authorization': get_access_token()}
+    request = requests.get(get_url(), params=query_params, headers=headers)
+    return request.status_code
```

Comment:
How about verifying that the links attribute is present in queries that return more than 10 results (along with links.self, links.next, links.prev)?